docs/observability/index.asciidoc

```diff
@@ -0,0 +1,24 @@
+[chapter]
+[role="xpack"]
+[[observability]]
+= Observability
+
+Observability enables you to add and monitor your logs, system
+metrics, uptime data, and application traces, as a single stack. 
+
+With *Observability*, you have:
+
+* A central place to add and configure your data sources.
+* A variety of charts displaying analytics relating to each data source.
+* *View in app* options to drill down and analyze data in the {logs-app}, {metrics-app}, {uptime-app}, and the APM app. 
```

Comment:
Suggest removing most uses of "app":

*View in app* options to drill down and analyze data in the Logs, Metrics, Uptime, and APM apps.